jobs/gtfs-rt-parser/gtfs_rt_parser.py

```diff
@@ -123,109 +216,359 @@ def identify_files(glob, rt_file_type: RTFileType, progress=False) -> List[RTFil
     # its caches (at least the ones I could find)
     fs.dircache.clear()
 
-    typer.secho(f"found {len(files)} files to process", fg=typer.colors.GREEN)
+    typer.secho(
+        f"found {len(files)} {rt_file_type} files in glob {glob}", fg=typer.colors.GREEN
+    )
     return files
 
 
+def download_gtfs_schedule_zip(gtfs_schedule_path, dst_path, pbar=None):
+    # fetch and zip gtfs schedule
+    zipname = f"{dst_path[:-1]}.zip"
+    log(
+        f"Fetching gtfs schedule data from {gtfs_schedule_path} to {zipname}",
+        pbar=pbar,
+    )
+
+    files = [
+        "agency.txt",
+        "calendar_attributes.txt",
```

Comment:
Ok I may have had a change of heart here (sorry)... I feel weird about us filtering this list at all. If you look in `cal-itp-data-infra.views.gtfs_schedule_dim_files` that should have a list of every file we've ever encountered, I kind of think we should stick with that. 

I actually think my preferred approach here would be to zip everything except what we know we put there: `validation.json` and the `processed/` subdirectory. I'm not sure how feasible it would be to do that, probably less performant but I think that that's like intellectually the correct thing.

Reply:
I think that's probably fair. I'll make the change.